extensions/http-curl/protocols/RESTSender.h

```diff
@@ -22,12 +22,13 @@
 #include "json/writer.h"
 #include <string>
 #include <mutex>
+
+#include "../../../libminifi/include/utils/ByteArrayCallback.h"
```

Comment:
Ditto.

Reply:
oh man sometimes eclipse really does this to me. thanks for pointing these out. I should really disable that feature